src/calendar.js

```diff
@@ -98,6 +109,7 @@ angular.module('ui.calendar', [])
       this.changeWatcher = function(arraySource, tokenFn) {
         var self;
         var getTokens = function() {
+          // console.log(arraySource);
           var array = angular.isFunction(arraySource) ? arraySource() : arraySource;
           var result = [], token, el;
           for (var i = 0, n = array.length; i < n; i++) {

```

Comment:
Travis fails here since test files where not updated. Therefore `scope.resources` is `undefined`.
I believe line 238 should be modified (see comment there)